terraform/environments/staging/portal.tf

```diff
@@ -349,7 +349,7 @@ module "domain" {
   source     = "../../modules/google-cloud/apps/elixir"
   project_id = module.google-cloud-project.project.project_id
 
-  compute_instance_type               = "n1-standard-2"
+  compute_instance_type               = "n1-standard-1"
```

Comment:
Just curious if this change was intentional? No worries if so, just double-checking.

Reply:
No worries.  Yeah, this is something @AndrewDryga asked me to put in since I was making TF changes.

@AndrewDryga, this is what you wanted me to change correct?

Reply:
Yeah, thank you. This is just to save money on staging.